submissions/ArthurGorbenko/html.css.practice/css/styles.css

```diff
@@ -0,0 +1,169 @@
+body{
+    font-family: arial,sans-serif;
+    font-size: 15px;
+}
+
+.container{
+    display: flex;
+    justify-content: flex-end;
+}
+
+div{
+    margin: 10px;
+}
+
+.content{
+    height: 500px;
+    width: 350px;
+    border: 1px solid #ccc;
+    float: right;
+    box-shadow: 0px 0px 10px 0px rgba(0,0,0,0.15);
+    overflow-y: auto;
+    display:none;  
+}
+
+ul{
+    list-style-type: none;
+    position: relative;
+    padding-inline-start: 20px;
+    margin-block-end:0px;
+}
+
+li{
+    display: inline-block;
+    height: 98px;
+    width: 86px;
+    padding: 5px;
+    border:1px solid rgb(204, 204, 204, 0);
+}
+
+li:hover{
+    border:1px solid rgb(204, 204, 204, 0.5);
+}
+
+a{
+    text-decoration: none;
+    position: absolute;
+    text-align: center;
+    margin: 10px;
+    position: relative;
+    color: black;
+}
+
+img{
+    height: 64px;
+    width: 64px;
+    display: inline-block;
+    vertical-align: top;
+}
+
+span {
+    text-align: center;
+    display: block;
+    width: 84px;
+}
+
+input{
```

Comment:
Navigating through TAB does not focus the menu button.
Try not to use display: none on your `checkbox`, instead, hide it using `clip: rect()`

Reply:
sadly, i can't fix that `label` for more button is focused first after open menu, it depepends on the relocation of `input`,but in the same time `input` must be located before content, what it hide on click.So, i guess current version is not dat bad, but i ready for tips